docs/features/techdocs/using-cloud-storage.md

```diff
@@ -38,14 +38,17 @@ files from here to serve documentation in Backstage. Note that the bucket names
 are globally unique.
 
 Set the config `techdocs.publisher.googleGcs.bucketName` in your
-`app-config.yaml` to the name of the bucket you just created.
+`app-config.yaml` to the name of the bucket you just created. Set
```

Comment:
Not sure this is the place to document it. Reading these makes me think setting it is required when it is in fact optional.

Reply:
Alright, let's get it in as is and we can iterate if needed.